CHANGELOG.asciidoc

```diff
@@ -3,6 +3,121 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.7.0]]
+=== Beats version 8.7.0
+https://github.com/elastic/beats/compare/v8.6.2\...v8.7.0[View commits]
+
+==== Breaking changes
+
+*Filebeat*
+
+- Metrics hosted by the HTTP monitoring endpoint for the `aws-cloudwatch`, `aws-s3`, `cel`, and `lumberjack` inputs are now available under `/inputs/` instead of `/dataset`.
```

Comment:
This is a feature not a breaking change. It was added in the wrong section.

The same is true for the two winlogbeat entries below.